index.html

```diff
@@ -21,30 +21,168 @@
     <section class="headline">
         <div id="bigText"><h1>I’m Adil
             Glad to see you!</h1></div>
-        <div id="smallText">
+        <div class="smallText">
             <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
         </div>
         <div class="socialmedia">
-            <div id="supportingText">LET'S CONNECT</div>
+            <div class="supportingText">LET'S CONNECT</div>
             <!--social media icons-->
             <ul>
             <li>
-                <a href="#"><img src="twitter.png" alt="twitter" class="icon icon1"></a>
+                <a href="#"><img src="twitter.png" alt="twitter"></a>
             </li>
             <li>
-            <a href="#"><img src="linkedin.png" alt="linkedin" class="icon icon2"></a>
+            <a href="#"><img src="linkedin.png" alt="linkedin"></a>
             </li>
             <li>
-            <a href="#"><img src="M.png" alt="M" class="icon icon3"></a>
+            <a href="#"><img src="M.png" alt="M"></a>
             </li>
             <li>
-            <a href="#"><img src="github.png" alt="github" class="icon icon4"></a>
+            <a href="#"><img src="github.png" alt="github"></a>
             </li>
             <li>
-            <a href="#"><img src="V.png" alt="V" class="icon icon5"></a>
+            <a href="#"><img src="V.png" alt="V"></a>
             </li>
             </ul>
         </div>
     </section>
+
+    <section>
+     <div class="grid-box">
+        <div class="box">
+            <img src="tonic.png" alt="Tonic" class="banner">
+            <div class="content">
+            <h1>Tonic</h1>
```

Comment:
`Comment 1:` You have to use `hx` heading for example `h2` or `h3`  for the heading.
This is because of the semantic/SEO reason. Search engines will get the `h1` as the title of the page. and for example `h2` and the main sections of the page, and `h3` as subsections, etc.
Kindly change the `h1` tag here and at all the next 4 cards to `h3`. But change the size of this ' h3` as per the size provided by Figma.